about us/about_us.js

```diff
@@ -0,0 +1,341 @@
+window.onload = () => {
+  renderCandidateTraits(candidateTraits);
+  renderProgramStageInfo(programStages);
+  renderKnownObstacles(knownObstacles);
+};
+
+const candidateTraitsBox = document.getElementById("candidate-traits");
+candidateTraitsBox.classList.add("candidate-traits");
+
+const candidateTraits = [
+  "Ability and willingness to focus on the transition for 6-24 month at least.",
+  "Full time starting with the “practice” phase.",
+  "Passion for problem solving: preferably in the form of science/engineering.",
+  "Analytic thinking.",
+  "Appetite to learn new applicable things continuously.",
+  "Pragmatism.",
+  "Will power and drive to persevere (given support and guidance).",
+];
+
+function renderCandidateTraits(candidateTraits) {
+  for (let candidateTrait of candidateTraits) {
+    const candidateTraitBox = document.createElement("div");
+    candidateTraitBox.classList.add("candidate-trait");
+    const candidateTraitDescription = document.createElement("p");
+    const checkSign = document.createElement("input");
+    checkSign.setAttribute("type", "checkbox");
+    checkSign.checked = true;
+    checkSign.classList.add("checkSign");
+
+    candidateTraitDescription.append(candidateTrait);
+    candidateTraitBox.append(checkSign);
+    candidateTraitBox.append(candidateTraitDescription);
+    candidateTraitsBox.append(candidateTraitBox);
```

Comment:
This works, but another way to do this will be to have a hidden element with the desired structure, i.e. a template. You can clone it and use when needed.